SparkPost/developers.sparkpost.com
# FE-1403 - Bumping high severity packages

### What Changed

- Bumped the below packages to resolve vulnerabilities
  - axios - plugins/gatsby-source-sparkpost-api/package-lock.json - **Upgraded to 0.21.1**
  - lodash - plugins/gatsby-source-sparkpost-api/package-lock.json - **Upgraded sparkpost to 2.1.4**
  - lodash - plugins/gatsby-transformer-api-blueprint/package-lock.json - **Upgraded lodash to 4.17.19**
  - lodash - package-lock.json
  - ua-parser-js - package-lock.json - **Upgraded styled-components**
  - y18n - package-lock.json - ** Upgrade from gatsby-plugin-algolia@0.11.1 and netlify-lambda@0.4.0 **
  - is-svg - package-lock.json - **Upgrade from gatsby@2.32.3**
  - url-parse - package-lock.json - **Upgrade from gatsby@2.32.3**
  - xmlhttprequest-ssl - package-lock.json - **Upgrade from gatsby@2.32.3**

- Upgrading of styled-components led to few changes in code - followed the guide here 👉  https://styled-components.com/docs/faqs#what-do-i-need-to-do-to-migrate-to-v4
- Upgraded Node version

### How to test

- Run npm list for above packages to check if you see any vulnerable version in the dependency list
- Check the [netlify deploy of the PR](https://deploy-preview-466--developers-sparkpost.netlify.app/) to verify if you see any discrepancy in styles because of styled-components upgrade
- Check the [netlify deploy of the PR](https://deploy-preview-466--developers-sparkpost.netlify.app/) to verify if you see anything broken because of the package upgrades


### To Do


- [ ] Couldn't Upgrade below packages - 

~- trim - package-lock.json - **Upgrade from gatsby@2.32.3 , gatsby-transformer-remark@2.8.25, netlify-cms@1.9.3, rehype-raw@2.0.0,  remark-parse@5.0.0 and remark-rehype@3.0.2**~

~- trim - plugins/gatsby-transformer-api-blueprint/package-lock.json  - **Upgraded remark to 9.0.0**~

~- ssri - package-lock.json - **Upgrade from webpack@4.46.0 and gatsby@2.32.3**~
